.ci/dev/Jenkinsfile

```diff
@@ -62,9 +71,18 @@ pipeline {
         expression { params.DO_PUBLISH }
         beforeAgent true
       }
+      environment {
+        S3_BUCKET=credentials("maven-s3-prod-bucket")
+        S3_PATH="s3://${env.S3_BUCKET}/maven/${env.CORDA_ARTIFACTORY_REPOKEY}"
+        AWS_CREDENTIALS=credentials("s3-prod-uploader")
+      }
       steps {
-        authenticateGradleWrapper()
-        sh "./gradlew --no-daemon -Pcompilation.allWarningsAsErrors=true -Ptests.failFast=false -Ptests.ignoreFailures=true artifactoryPublish"
+        sh '''./gradlew publish artifactoryPublish \
```

Comment:
Where does this extra `publish` task come from?

Reply:
@chrisr3 that's correct.

It's provided by `maven-publish plugin?`

_"An aggregate task that publishes all defined publications to all defined repositories. It does not include copying publications to the local Maven cache."_

So in my case, here it just pushes to the defined repo i added in build.gradle